dask/dataframe/tests/test_dataframe.py

```diff
@@ -2728,6 +2728,20 @@ def test_applymap():
     assert_eq(ddf.applymap(lambda x: (x, x)), df.applymap(lambda x: (x, x)))
 
 
+def test_add_prefix():
+    df = pd.DataFrame({"x": [1, 2], "y": [3, 4]})
+    ddf = dd.from_pandas(df, npartitions=2)
+    assert_eq(ddf.add_prefix("abc"), df.add_prefix("abc"))
+    assert_eq(ddf.x.add_prefix("abc").compute(), df.x.add_prefix("abc"))
```

Comment:
hmm, that's what I had done initially but the test failed. I'll try again!

Reply:
See the failing test when I don't call `compute` - I suspect it's because we change the index to `str`s and `assert_eq` does some checks/manipulatiosn on the indices that assume they are `int`s (or something like that, though in that case, shouldn't it fail regardless of calling `.compute()`?).

Feel free to revert the latest commit. I split them into 2 in case one failed!